KhushSharma_KaprekarConstantAssignmentz.java

```diff
@@ -0,0 +1,81 @@
+import java.util.Arrays;
+import java.util.Scanner;
+import java.util.InputMismatchException;
+
+class KaprekarConstant {
+
+	private static int findKaprekarRoutine(int number) {
+
+		int iterations = 0;
+		int numberOfDigits = (Integer.toString(number)).length();
+
+		int[] arrayOfDigits = new int[numberOfDigits];
+
+		while (number != 6174) {
+			for (int i = numberOfDigits - 1; i >= 0; i--) {
```

Comment:
This loop code is a different functionality. So, It can be converted into a separate function.